components/rmrk/Gallery/History.vue

```diff
@@ -275,6 +275,7 @@ export default class History extends mixins(ChainMixin, KeyboardEventsMixin) {
         priceCollectorMap[nftId] = '0'
       } else if (newEvent['interaction'] === 'BUY') {
         event['Type'] = this.$t('nft.event.BUY')
+        priceCollectorMap[nftId] = newEvent['meta']
```

Comment:
Why we need this?

Reply:
solve '-' issue. #3027

Reply:
I do not understand this. Maybe ELI5?

Reply:
Hmm, sorry, could elaborate ELI5 , it means?

Reply:
Explain it like I am 5-year-old person

Reply:
It's irrelevant with refactoring, just free ride to solve a bug roiLee found.

👌 the issue I mentioned before, it does not show the money.

The reason is we did not add money data to priceCollectorMap when event is 'buy'.

Reply:
OK :D but still confused why we do not use meta from the buy event ?

Reply:
<img width="676" alt="image" src="https://user-images.githubusercontent.com/11361147/169649678-1fb774b0-81f6-4e90-a370-70d9cc032ca5.png">

here is the original logic.

Reply:
if for `Buy` event we do not set `priceCollectorMap` for nft, it would return `undefined`, then the ternary would go to `'-'`